redisinsight/ui/src/pages/rdi/pipeline/components/header/RdiPipelineHeader.spec.tsx

```diff
@@ -12,21 +14,49 @@ beforeEach(() => {
   store.clearActions()
 })
 
+jest.mock('uiSrc/telemetry', () => ({
+  ...jest.requireActual('uiSrc/telemetry'),
+  sendEventTelemetry: jest.fn(),
+}))
+
 jest.mock('uiSrc/slices/rdi/instances', () => ({
   ...jest.requireActual('uiSrc/slices/rdi/instances'),
   connectedInstanceSelector: jest.fn().mockReturnValue({
     name: 'name',
   }),
 }))
 
+jest.mock('uiSrc/slices/rdi/pipeline', () => ({
+  ...jest.requireActual('uiSrc/slices/rdi/pipeline'),
+  rdiPipelineSelector: jest.fn().mockReturnValue({
+    loading: false,
+  }),
+}))
+
 jest.mock('react-router-dom', () => ({
   ...jest.requireActual('react-router-dom'),
   useHistory: () => ({
     push: jest.fn,
   }),
 }))
 
+jest.mock('formik')
+
+beforeEach(() => {
+  cleanup()
+})
```

Comment:
do we need this cleanup() one more time?

Reply:
I guess nope. Removed. Thank you